source/components/dateTime/dateTime.ts

```diff
@@ -159,7 +159,16 @@ function dateTime(moment: moment.MomentStatic
 			let max: string | Date | moment.Moment
 				= dateTime.max != null ? dateTime.max : defaults.maxDate;
 
-			scope.$watch((): any => { return ngModel.$viewValue; }, (newValue: any): void => {
+			ngModel.$formatters.push((value: moment.Moment): string => {
+				return moment(value).format(getFormatOrDefault());
```

Comment:
Again: interesting that you are calling `moment(value)` on an object that, by the typings, appears to already be a `moment` object... ?